engines/sci/graphics/text32.h

```diff
@@ -362,15 +367,15 @@ class GfxText32 {
 
 	/**
 	 * The size of the x-dimension of the coordinate system
-	 * used by the text renderer.
+	 * used by the text renderer. NB: static
```

Comment:
I think the static keyword in the code itself means it is not necessary to add extra comments about it being static :)